lib/footnotes.js

```diff
@@ -24,24 +24,22 @@ function setRef(link)
 
 function processPage(page)
 {
-  var options = this.config.options
+  if(this.output.name !== 'ebook') return page
 
-  if(options.generator !== 'pdf') return page
-
-  var language = intl[options.language || 'en']
+  var i10nPrefix = intl[this.config.get('language')] || intl['en']
```

Comment:
if it would be done as before and `options.language` has a value that's not defined in `intl.json`, `language` will be `undefined`.
-> the default will not work